src/functional/geometry/convex_hull.zig

```diff
@@ -25,14 +25,15 @@ const ArrayList = std.ArrayList;
 const testing = std.testing;
 
 const tersets = @import("../../tersets.zig");
-const Point = tersets.Point;
+const Point = tersets.DiscretePoint;
```

Comment:
Maybe replace with `const Point` with `const DiscretePoint` like in `swing_slide_filter.zig` to make it explicit which version is used and make the code consistent.

Reply:
Fixed.